install/installer/pkg/components/database/incluster/init/04-drop-sessions-db.sql

```diff
@@ -0,0 +1,10 @@
+-- Copyright (c) 2020 Gitpod GmbH. All rights reserved.
+-- Licensed under the GNU Affero General Public License (AGPL). See License.AGPL.txt in the project root for license information.
+
+-- must be idempotent
+
+USE `gitpod-sessions`;
+
+DROP TABLE IF EXISTS `sessions`;
```

Comment:
I think we should remove [this](https://github.com/gitpod-io/gitpod/blob/478a75e744a642d9b764de37cfae655bc8b29dd5/install/installer/pkg/components/database/init/files/00-create-and-init-sessions-db.sql#L10-L16) and [this](https://github.com/gitpod-io/gitpod/blob/478a75e744a642d9b764de37cfae655bc8b29dd5/install/installer/pkg/components/database/incluster/init/01-create-and-init-sessions-db.sql#L10-L16) before running these migrations. :+1:

Reply:
In other words, I think the existing migrations need to stay unaltered as they are immutable once applied